CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
-## Unreleased (v3.1.x)
+## All versions staring in 2022
+A changelog is maintained on the releases page of the [XUnit Test Logger GitHub repository](https://github.com/spekt/xunit.testlogger/).
```

Comment:
@Siphonophora there's one challenge to this: if we do infrequent releases, e.g. every quarter or so, it becomes a hard task to go through all changes and collate those :( Changelog.md solves this in a rudimentary way. Curious what are the challenges with changelog.md?

Reply:
Oh, do you accumulate items under the unreleased tag as you go? That makes sense. I was planning to use the release page b/c we don't know the release version number until after it's built.

I'm happy to do that here.

Reply:
Yes, we keep collecting the changes incrementally :)

Reply:
All set